src/realm/sync/client.cpp

```diff
@@ -868,8 +881,13 @@ bool SessionImpl::process_flx_bootstrap_message(const SyncProgress& progress, Do
     REALM_ASSERT_EX(hook_action == SyncClientHookAction::NoAction, hook_action);
 
     if (batch_state == DownloadBatchState::MoreToCome) {
+        notify_download_progress(bootstrap_store->pending_stats().pending_changeset_bytes);
         return true;
     }
+    else {
+        // FIXME this shouldn't be needed anymore when progress is reported separately for every direction
```

Comment:
is this comment correct? I guess we need to reset it somewhere (I would do it when the progress gets to 1.0)

Reply:
yes, this is correct. It's possible to remove this variable completely and simply pass it through notify_download_progress as a temporary but it's really only possible with solving #7451. I've rephrased this fixme to better communicate the intent here.
Also, this is essentially the place in code where it's known that we don't get more changesets for bootstrap store, so i'd like to keep it here to not accidentally report doubled bytes.